add methanol-to-kerosene cost data

p-glaum opened the pull request:
Closes # (if applicable).

## Changes proposed in this Pull Request


## Checklist

- [ ] Code changes are sufficiently documented; i.e. new functions contain docstrings and further explanations may be given in `doc`.
- [x] Data source for new technologies is clearly stated.
- [ ] Newly introduced dependencies are added to `environment.yaml` (if applicable).
- [x] A note for the release notes `doc/release_notes.rst` of the upcoming release is included.
- [x] I consent to the release of this PR's code under the GPLv3 license.

fneum reviewed (approved):
LGTM, release note to be added.

p-glaum commented:
> @p-glaum can you also update the output files? Is there also data for other years than 2030 in the paper? If so, it would be nice to have it in the technology data as well.

hi @lisazeyen, I updated the costs.csv files and added the cost years provided in the report.

lisazeyen commented:
Great, thanks @p-glaum !
